stdlib/private/StdlibCollectionUnittest/CheckMutableCollectionType.swift.gyb

```diff
@@ -988,13 +992,18 @@ self.test("\(testNamePrefix).partition/DispatchesThrough_withUnsafeMutableBuffer
 
 %   if predicate:
   let closureLifetimeTracker = LifetimeTracked(0)
-  let pivot = lc.partition() {
-    (lhs, rhs) in
-    _blackHole(closureLifetimeTracker)
-    return extract(lhs).value < extract(rhs).value
+  var pivot = lc.startIndex
+  if let first = c.first {
+    pivot = lc.partition(by: {
+      val in
+      _blackHole(closureLifetimeTracker)
+      return !(extract(val).value < extract(first).value) })
   }
 %   else:
-  let pivot = lc.partition()
+  var pivot = lc.startIndex
+  if let first = lc.first {
+    pivot = lc.partition(by: { !($0 < first) })
+  }
```

Comment:
Same, if there is no `partition` without the predicate, please drop this branch.